Tests/FluentAssertions.Specs/Equivalency/DataEquivalencySpecs.cs

```diff
@@ -0,0 +1,4632 @@
+        [SuppressMessage("Style", "IDE0010:Add missing cases", Justification = "Every enum value is covered")]
+        private static void ApplyChange(DataColumnCollection columns, ChangeType changeType)
+        {
+            switch (changeType)
+            {
+                case ChangeType.Added:
+                    columns.Add("Test", typeof(int));
+                    break;
+                case ChangeType.Changed:
+                    columns[1].ColumnName += "different";
+                    break;
+                case ChangeType.Removed:
+                    columns.RemoveAt(1);
+                    break;
+            }
+        }
+
+        [SuppressMessage("Style", "IDE0010:Add missing cases", Justification = "Every enum value is covered")]
+        private static void ApplyChange(PropertyCollection extendedProperties, ChangeType changeType)
+        {
+            switch (changeType)
+            {
+                case ChangeType.Added:
+                    extendedProperties[Guid.NewGuid()] = Guid.NewGuid();
+                    break;
+                case ChangeType.Changed:
+                    extendedProperties[extendedProperties.Keys.Cast<object>().First()] = Guid.NewGuid();
+                    break;
+                case ChangeType.Removed:
+                    extendedProperties.Remove(extendedProperties.Keys.Cast<object>().First());
+                    break;
+            }
+        }
+
+        [SuppressMessage("Style", "IDE0010:Add missing cases", Justification = "Every enum value is covered")]
+        private static void ApplyChange(ConstraintCollection constraints, DataColumn columnForNewConstraint, ChangeType changeType)
+        {
+            switch (changeType)
+            {
+                case ChangeType.Added:
+                    constraints.Add(
+                        new UniqueConstraint(
+                            "Test",
+                            columnForNewConstraint));
+                    break;
+                case ChangeType.Changed:
+                    constraints[1].ConstraintName += "different";
+                    break;
+                case ChangeType.Removed:
+                    constraints.RemoveAt(1);
+                    break;
+            }
+        }
+
+        [SuppressMessage("Style", "IDE0010:Add missing cases", Justification = "Every enum value is covered")]
+        private static void ApplyChange(DataRowCollection rows, DataTable dataTable, ChangeType changeType)
+        {
+            switch (changeType)
+            {
+                case ChangeType.Added:
+                    InsertDummyRow(dataTable);
+                    break;
+                case ChangeType.Changed:
+                    rows[1]["String"] += "different";
+                    break;
+                case ChangeType.Removed:
+                    rows.RemoveAt(0);
+                    break;
+            }
+        }
+
+        public class DataSetEquivalencySpecs : DataEquivalencySpecs
```

Comment:
If possible, please split out the subclasses of `DataEquivalencySpecs` to separate files.